rudder-core/src/main/scala/com/normation/rudder/services/policies/write/Cf3PromisesFileWriterService.scala

```diff
@@ -356,6 +356,7 @@ class Cf3PromisesFileWriterServiceImpl(
          * and "error/technical details"
          */
         val completeErrorMsg = ( s"The generated promises are invalid!errormessage!cf-promise check fails for promises generated at '${paths.newFolder}'"
+                               + "<-Test command is: '" + checkCommand + "'"
```

Comment:
I'm wondering if the message could be better with "promise check command is:" 
What do you think @jooooooon ?